<launch>
	<node pkg="mavel" type="pose_odom_node" name="pose_odom" clear_params="true" output="screen">
		<param name="frame_world" value="world" type="str"/>
		<param name="frame_mav" value="fcu" type="str"/>

		<param name="topic_odom" value="/odom" type="str"/>
		<param name="topic_pose" value="/mavros/local_position/pose" type="str"/>

		<param name="velocity_filtering" value="0.4" type="double"/>
	</node>
</launch>
